packages/@aws-cdk/aws-apigateway/test/authorizers/integ.request-authorizer-iam-role.ts

```diff
@@ -0,0 +1,47 @@
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
Given that IAM set up is now part of the base class (`LambdaAuthorizer`), one instance of this (i.e. `integ.token-authorizer-iam-role.ts`) is sufficient. 

Feel free to drop this one, however, it's not an approval blocker.